src/UCommerce.SiteCore.Installer/Steps/MoveResourceFiles.cs

```diff
@@ -58,15 +59,15 @@ public Task Run()
 
             foreach (var file in files)
             {
-                postInstallationSteps.Add(new MoveFile(new FileInfo(Path.Combine(SourceDirectory.FullName, file)),
+                steps.Add(new MoveFile(new FileInfo(Path.Combine(SourceDirectory.FullName, file)),
                     new FileInfo(Path.Combine(TargetDirectory.FullName, file)),
                     false,
                     _loggingService));
             }
 
-            foreach (var postInstallationStep in postInstallationSteps)
+            foreach (var step in steps)
             {
-                postInstallationStep.Run();
+                step.Run();
             }
 
             return Task.CompletedTask;

```

Comment:
These steps are async are they not?

So we could just use async/await in this method instead of returning the Task.CompletedTask?

I know this was not changed in your PR but I think we need to fix it anyways

Reply:
Of course! Missed that